src/main/java/frc/robot/subsystems/Turret.java

```diff
@@ -22,6 +24,9 @@
   private WPI_TalonSRX mTurretTalon;
   private Potentiometer mPot;
 
+  // TODO: this
+  private Relay cameraLight = new Relay(Constants.cameraRingId);
```

Comment:
We aren't using config for this?

Reply:
I'm gonna work on that in #52 so you can ignore it @onedeadeye

Reply:
@onedeadeye > heard